### PR TITLE
New Test for Vijar

### PR DESCRIPTION
**Pull request by grinred**
Look and check it, someone, please!

### CODE REVIEW

File: src/test/java/RozetkaSergey/Vijar.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import org.testng.annotations.Test;
+public class Vijar extends TestInit {
+
+String login = "grinred69@gmail.com";
+String pass = "23112010vi";
+
+    @Test
+    public void loginTest() {
+        enterToGoogle();
+        sleep(1);
```
Comment (StepaniukAndii) [inline src/test/java/RozetkaSergey/Vijar.java:15]: sleep it's bad practice. Best Explicit Wait

File: src/test/java/RozetkaSergey/Vijar.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        return driver.findElement(By.xpath("//input[@class='flat_input login']"));
+    }
+    public WebElement getPass(){
+        return driver.findElement(By.xpath("//input[@class='flat_input password']"));
+    }
+    public WebElement clickEnter() {
+        return driver.findElement(By.xpath("//input[@class='login_btn']"));
+    }
+}
+
```
Comment (StepaniukAndii) [inline src/test/java/RozetkaSergey/Vijar.java:60]: in new class

File: src/test/java/RozetkaSergey/Vijar.java (hunk 1 above)
Comment (grinred) [reply src/test/java/RozetkaSergey/Vijar.java:15]: ok I will do it tommorow

Comment (StepaniukAndii) [reply src/test/java/RozetkaSergey/Vijar.java:15]: ok)

**Comment by grinred** [comment]
I know that it isn't ready!(

**Pull request closed without merging**